test/integration/edition_edit_test.rb

```diff
@@ -157,5 +183,51 @@ class EditionEditTest < IntegrationTest
         assert page.has_link?("Cancel")
       end
     end
+
+    context "admin tab" do
+      setup do
+        click_link("Admin")
+      end
+
+      should "show 'Admin' header and not show 'Skip fact check' button" do
+        within :css, ".gem-c-heading" do
+          assert page.has_text?("Admin")
+        end
+        assert page.has_no_button?("Skip fact check")
+      end
+    end
+  end
+
+  context "when edition state is 'fact_check'" do
```

Comment:
There's tests here for the draft, published and out for fact check states, but what about the other states?